.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "mainline-1.x"
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "mainline-2.x"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Do we know if dependabot will make recommendations for *any* requirements file? Or will it only think about the requirements file in root directory and ignore `test/requirements.txt`?

Reply:
It'll look at any requirements file, it's the same way with the ddbec-java repo. There are two pom.xml files, one in the root and one in sdk1/